### DISCUSSION TITLE
how to enable "allowDiskUse" in aggregate

### DISCUSSION THREAD

**Discussion by sky-cloud** [mongo-express/mongo-express#1055] [Q&A]
The memory limit of aggregate opertation is 100 MB be default, which is inconvenient to view large collections. How to set the `allowDiskUse` property of `AggregateOperation` to `true`? 

**Environments**: MongoDB 4.4.18, mongo-express build from node:18-alpine3.16. Both were deployed by docker 20.10.8, docker-compose 1.28.2.

Ref: [Aggregation Pipeline Limits](https://www.mongodb.com/docs/v4.4/core/aggregation-pipeline-limits/)

![Selection_070](https://user-images.githubusercontent.com/9923507/203896582-dab942c2-583f-4d9c-9870-079eedd8b602.png)

**Comment by rtritto** [comment]
Similar issue #1086
Related issue #1089
